ocs_sample_library_preview/DataView/FieldKind.py

```diff
@@ -3,8 +3,10 @@
 
 class FieldKind(Enum):
     """
-    enum 0-2  fully inclusive
+    enum 0-4  fully inclusive
     """
     IndexField = 0
     GroupingField = 1
```

Comment:
looks like commas are option in the enum definition list, but we should either trail 0 and 1 with a comma, or remove them from 2 and 3

Reply:
Trailing comma actually breaks the enum values (it keeps the comma as part of the value) so I removed these.